app/models/survey.rb

```diff
@@ -7,11 +7,14 @@
 #  description                          :text
 #  created_at                           :datetime         not null
 #  updated_at                           :datetime         not null
+#  first_question_id                    :integer
 #
 
 class Survey < ActiveRecord::Base
   has_many :question_orders
   has_many :survey_responses
 
-  validates :description, :name, presence: true
+  belongs_to :first_question, foreign_key: :first_question_id, class_name: 'Question'
+
+  validates :description, :name, :first_question_id, presence: true
```

Comment:
okaaay true, I was thinking we wouldn't allow them to create a survey without any questions but I guess why not